### PR TITLE
Add draft holiday cancellation guide

### PR DESCRIPTION
**Pull request by jcoliver**
A draft of practice to use for cancelling trainers meeting for discussion at June Trainers leadership meeting (addresses #93 ).

Going to add everyone as a reviewer, although not sure that's the right mechanism for adopting this thing. :thinking:

### CODE REVIEW

**Comment by brownsarahm** [comment]
A challenge with 24h notice is that a trainer could be relying on the meeting for a pre-workshop discussion; what could we do to support people in that?

**Comment by karenword** [comment]
Thanks @jcoliver ! Per Sarah's comment, I think these last minute changes are a little hard on everyone and it would be nice to plan in advance, though of course we should always leave room for last minute changes. I also wonder how Trainers are going to let me know. Maybe I can set up a request form and bump it a couple of times a year, with a list of upcoming meeting dates to object to? And if something still comes up at the last minute, they can use the form so that those holidays can be planned for in the future?

**Comment by Denubis** [comment]
Question: Since nationalities and holidays are public data (and I'm reminded of the good Friday cancellation a few months ago that the Aussies went 'but it's a holiday!' -- isn't it easier to use nationalities from github, apply the "holiday template" to the meetings calendar and go from there? Adding another response required for known information sounds a bit tedious to me.

**Comment by karenword** [comment]
It is possible that we could determine nationality for all Trainers and use official public holidays -- this indeed would have addressed the most recent problem. However, there may be widely celebrated religious holidays that are not state holidays, for example. Also, as our community expands, it may not be practical to cancel meetings for *every* state holiday in *every* country where we have a Trainer. I think this calls for one of those fuzzy middle points, which is a good reason to create a policy (and change it as needed).

File: policy/holiday-scheduling-process.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+major holiday, reducing participation by the trainers community. Given the 
+global distribution of Carpentries Trainers, and variation in observance of 
+major holidays, the degree to which any particular holiday will impact trainer 
+meeting collision is difficult to predict. 
+
+## Responsibilities
+
++ The decision to cancel a regularly scheduled trainers meeting will be the 
+responsibility of the Carpentries' Director of Instructor Training. 
++ Notification of such a cancellation should be made at least 24 hours before the
```
Comment (marklcrowe) [inline policy/holiday-scheduling-process.md:15]:
```suggestion
+ Notification of such a cancellation should be made at least 72 hours before the
```

**Review by marklcrowe — CHANGES_REQUESTED** [review]
This discussion seems to be getting a bit involved. Isn't the ideal for this request basically "when should we cancel a trainer meeting and how much notice is required" rather than developing complex systems for identifying public holidays and the like. 

I'm in favour of the initial suggestion - that the community itself is responsible for flagging any significant clashes. And this doesn't in itself mean that a meeting is cancelled, especially if someone is relying on it for a pre-workshop meeting (do people register that beforehand so we'd know?); that again is covered in the proposal, it being at the discretion of the Director of Training. Neither of those really should be a last minute thing so I'd prefer that the notice period was a bit longer, ideally 72h (48 would be OK).

tl;dr - I approve this proposal if we extend the cancellation time out to 72h

File: policy/holiday-scheduling-process.md (hunk 1 above)
Comment (arinbasu) [inline policy/holiday-scheduling-process.md:15]: I think a 24 hour notice is too early, perhaps a 72 hour window is better

**Review by arinbasu — DISMISSED** [review]

**Comment by jcoliver** [comment]
The draft policy is explicit in responsibility, but vague in conditions _why_ a meeting may be cancelled. The legion of holidays our trainer community experiences is diverse, and while we might be able to track holidays, cancellation because of a holiday should only happen when that holiday would influence a significant* portion of the Trainers community that would otherwise attend a Trainers meeting (i.e. even if National Doughnut Day (either of them) were to fall on a Trainers meeting day, it remains an open question of how many Trainers would skip the Trainers meeting to celebrate the glory of leavened fried dough).

To address questions of how cancellation might effect pre-training discussions, we could add a line to the effect of "The Director of Instructor Training should attempt to accommodate meetings when novice instructors (< 1year) have signed up for pre-workshop discussions"

*Recognizing, of course, that "significant" is arbitrary.

File: policy/holiday-scheduling-process.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+Regularly scheduled bi-monthly meetings of Carpentries Trainers may fall on a 
+major holiday, reducing participation by the trainers community. Given the 
+global distribution of Carpentries Trainers, and variation in observance of 
+major holidays, the degree to which any particular holiday will impact trainer 
+meeting collision is difficult to predict. 
+
+## Responsibilities
+
++ The decision to cancel a regularly scheduled trainers meeting will be the 
+responsibility of the Carpentries' Director of Instructor Training.
```
Comment (brownsarahm) [inline policy/holiday-scheduling-process.md:14]:
```suggestion
+ The decision to cancel a regularly scheduled trainers meeting will be at the 
discretion of the Carpentries' Director of Instructor Training or a designated proxy. 
```

proxy meaning like, if the director was out of office for some reason another person could cancel per whatever  arrangement for coverage was made by the director

File: policy/holiday-scheduling-process.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+global distribution of Carpentries Trainers, and variation in observance of 
+major holidays, the degree to which any particular holiday will impact trainer 
+meeting collision is difficult to predict. 
+
+## Responsibilities
+
++ The decision to cancel a regularly scheduled trainers meeting will be the 
+responsibility of the Carpentries' Director of Instructor Training. 
++ Notification of such a cancellation should be made at least 72 hours before the
+scheduled meeting time.
```
Comment (brownsarahm) [inline policy/holiday-scheduling-process.md:16]:
```suggestion
scheduled meeting time through all normal meeting announcement communication channels. 
```

File: policy/holiday-scheduling-process.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+## Responsibilities
+
++ The decision to cancel a regularly scheduled trainers meeting will be the 
+responsibility of the Carpentries' Director of Instructor Training. 
++ Notification of such a cancellation should be made at least 72 hours before the
+scheduled meeting time. 
++ The Carpentries Trainers community will be responsible for notifying the 
+Director of Instructor Training of significant holidays that conflict with 
+regularly scheduled trainer meetings.
```
Comment (brownsarahm) [inline policy/holiday-scheduling-process.md:19]:
```suggestion
+ Any member of the Carpentries Trainers community may notify the 
Director of Instructor Training of significant holidays that conflict with 
regularly scheduled trainer meetings. 
```
also maybe, logically this bullet should be first?

**Comment by brownsarahm** [comment]
I also like @jcoliver 's suggestion about new trainer pre/post workshops.  

Maybe we say that new discussion topics can be postponed at the shorter (72h) notice, but that canceling time for pre/post discussions has to be done further in advance maybe by the preceding meeting?  For holidays where there's no one to facilitate level impact it's feasible that the director can update the calendar a few months in advance.  For things that impact fewer people, but still enough that we want to postpone important discussions in favor of greater inclusion the short notice should be okay.

File: policy/holiday-scheduling-process.md (hunk 1 above)
Comment (jcoliver) [reply policy/holiday-scheduling-process.md]: Indeed. Updated to 72 hours.

**Review by ACharbonneau — APPROVED** [review]

**Comment by jcoliver** [comment]
@jcoliver, FLTOG, add that piece about accommodating pre/post June 11: "The Director of Instructor Training should attempt to accommodate meetings when novice instructors (< 1year) have signed up for pre-workshop discussions"

**Review by ACharbonneau — APPROVED** [review]

File: policy/holiday-scheduling-process.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+discretion of the Carpentries' Director of Instructor Training or a designated 
+proxy. 
++ Notification of such a cancellation should be made at least 72 hours before 
+the scheduled meeting time through all normal meeting announcement communication 
+channels. 
++ Any member of the Carpentries Trainers community may notify the 
+Director of Instructor Training of significant holidays that conflict with 
+regularly scheduled trainer meetings. 
++ The Director of Instructor Training should attempt to accommodate meetings 
+when novice instructors (< 1year) have signed up for pre-workshop discussions
```
Comment (brownsarahm) [inline policy/holiday-scheduling-process.md:23]:
```suggestion
when novice trainers (< 1year) have signed up for pre-workshop discussions
```

**Comment by jcoliver** [comment]
Thanks for the feedback, folks. @marklcrowe , I think this needs your stamp of approval to move forward.

**Review by brownsarahm — APPROVED** [review]

**Review by marklcrowe — APPROVED** [review]
